flink-end-to-end-tests/flink-end-to-end-tests-common-kafka/pom.xml

```diff
@@ -216,14 +216,31 @@ under the License.
 						<artifactItem>
 							<groupId>org.apache.kafka</groupId>
 							<artifactId>kafka-clients</artifactId>
-							<version>2.4.1</version>
+							<version>2.8.1</version>
 							<destFileName>kafka-clients.jar</destFileName>
 							<type>jar</type>
 							<outputDirectory>${project.build.directory}/dependencies</outputDirectory>
 						</artifactItem>
 					</artifactItems>
 				</configuration>
 			</plugin>
+
+			<!-- Skip dependency convergence check because of guava version -->
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-enforcer-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>dependency-convergence</id>
+						<goals>
+							<goal>enforce</goal>
+						</goals>
+						<configuration>
+							<skip>true</skip>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
```

Comment:
I've based this on the same implementation as in `flink-end-to-end-tests/flink-streaming-kinesis-test/pom.xml`

Reply:
@zentol although we also used it in kinesis for some reason what would be the right way?

Reply:
why not just fix the convergence issue?

Reply:
I don't know why we skip convergence for kinesis. It doesn't sound like a good idea to me because guava might be bundled, and there we should take care we pick the right one...

Reply:
I've excluded `com.google.guava:guava` from `org.apache.flink:flink-connector-testing` in `flink-end-to-end-tests/flink-end-to-end-tests-common-kafka/pom.xml` and removed the convergence check. The CI passed locally, so let's see if this resolves it.